package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "wikibase-datamodel-javascript",
+  "version": "3.0.1",
+  "description": "Wikibase datamodel implementation in JavaScript",
+  "directories": {
+    "lib": "src",
+    "test": "tests"
+  },
+  "dependencies": {
+    "jquery": "^3.2.1"
+  },
+  "devDependencies": {
+    "karma": "^1.7.0",
+    "karma-cli": "^1.0.1",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-qunit": "^1.2.1",
+    "qunitjs": "^2.3.2"
+  },
+  "scripts": {
+    "postinstall": "composer install",
+    "test": "node_modules/.bin/karma start --single-run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wmde/WikibaseDataModelJavaScript.git"
+  },
+  "author": { "name" : "H. Snater", "url" : "http://www.snater.com"},
+  "contributors":[
+    { "name" : "H. Snater", "url" : "http://www.snater.com"},
+    { "name" : "Adrian Lang", "email" : "adrian.lang@wikimedia.de"}
```

Comment:
I am not sure if it makes sense to include this email address when Adrian is no longer working for WMDE. Is it OK to only have a "name" here?

For probably more important part: I guess would be better to "title" Adrian as "Adrian Heine".